projects/angular/src/data/datagrid/API.md

```diff
@@ -0,0 +1,53 @@
+| --clr-datagrid-placeholder-font-size                  | Changes the placeholder text size of datagrid.                               |
+| --clr-datagrid-pagination-btn-disabled-opacity        | Changes the opacity of pagination button when disabled.                      |
+| --clr-datagrid-column-toggle-border-color             | Changes the column toggle button border color.                               |
+| --clr-datagrid-column-toggle-fill-color               | Changes the background color of column toggle button.                        |
+| --clr-datagrid-column-toggle-text-color               | Changes the text color of column toggle button.                              |
+| --clr-datagrid-column-toggle-border-hover-color       | Changes the border color of toggle button when hovered over.                 |
+| --clr-datagrid-column-toggle-fill-hover-color         | Changes the background color of column toggle button when hovered over.      |
+| --clr-datagrid-column-toggle-text-hover-color         | Changes the text color of toggle button when hovered over.                   |
+| --clr-datagrid-column-toggle-border-active-color      | Changes the border color of toggle button when active.                       |
+| --clr-datagrid-column-toggle-fill-active-color        | Changes the background color of toggle button when active.                   |
+| --clr-datagrid-column-toggle-text-active-color        | Changes the text color of pagination toggle button when active.              |
+
+
+## Class names
+
+| Class name                            | Description                                                                     |
+| ------------------------------------- | -----------------------------------------------------------------------------   |
+| .clr-datagrid                         | Identify the main datagrid element.                                             |
+| .datagrid-nav                         | To filter and navigate through data displayed in datagrid.                      |
+| .datagrid-outer-wrapper               | It's container element that holds and positions data grid component.            |
+| .datagrid-calculation-table           | Allows for the display and manipulation of calculated data.                     |
+| .datagrid-host                        | To manage and optimize the performance of datagrid.                             |
+| .datagrid-header                      | To displays the column labels or headers.                                       |
+| .datagrid-row                         | To display single record or item of data in a horizontal format.                |
+| .datagrid-row-master                  | Primary row that used to group or summarize related data in hierarchical format.|
+| .datagrid-row-flex                    | Allows for flexible sizing and positioning of cells within data grid.           |
```

Comment:
Internal datagrid row container, managing "flex" positioning

Reply:
Changed as you suggested.